hwcomposer/hwcomposer.cpp

```diff
@@ -1006,6 +1006,9 @@ static int hwc_open(const struct hw_module_t* module, const char* name,
     if (!property_get_bool("persist.waydroid.cursor_on_subsurface", false))
         pdev->display->cursor_surface =
             wl_compositor_create_surface(pdev->display->compositor);
+    else
+        pdev->display->cursor_surface = nullptr;
```

Comment:
I'm not really well-versed in C++, but since the display is allocated with `struct display *display = new struct display();`, this should mean that all its members get value-initialized, which in case of pointers initializes them to `nullptr` already. Is that not the case here?